github/README.md

```diff
@@ -34,6 +34,7 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/githu
 
 * **REPO_FULL_NAME:**: The GitHub repository full name, _e.g:_ `tektoncd/catalog`
 * **GITHUB_HOST_URL:**: The GitHub host domain _default:_ `api.github.com`
+* **API_PATH_PREFIX:**: The GitHub Enterprize has a prefix for the API path. _e.g:_ `/api/v3`
```

Comment:
`Enterprize` -> `Enterprise`

(also lines 96 & 149)

Reply:
Thanks. I fix it.